src/java.base/share/classes/jdk/internal/foreign/abi/aarch64/linux/LinuxAArch64Linker.java

```diff
@@ -41,13 +42,16 @@
  * the ARM 64-bit Architecture".
  */
 public final class LinuxAArch64Linker extends AbstractLinker {
-    private static LinuxAArch64Linker instance;
+    private static final class Holder {
+        private static final LinuxAArch64Linker INSTANCE = new LinuxAArch64Linker();
+    }
```

Comment:
You could also make this a local class in `getInstance` (they now allow static fields).